mteb/abstasks/AbsTaskReranking.py

```diff
@@ -5,10 +5,11 @@
 class AbsTaskReranking(AbsTask):
     """
     Abstract class for re-ranking experiments.
-    Child-classes must implement the following properties:
-    self.corpus = {'dev': Dict[id, str], 'test': Dict[id, str]}         #id => sentence
-    self.queries = {'dev': Dict[id, str], 'test': Dict[id, str]}
-    self.relevant_docs = {'dev': Dict[id, set], 'test': Dict[id, set]}
+
+    self.load_data() must generate a huggingface dataset with a split matching self.description["eval_splits"], and assign it to self.dataset. It must contain the following columns:
+        query: str
+        positive: list[str]
+        negative: list[str]
```

Comment:
Not sure about this one either, but if you are then I am fine with it. Looks like the old docstring is def. wrong.